### PR TITLE
styling_BZ

### PR DESCRIPTION
**Pull request by bartzaw**
I've removed some unecessary stuff in my sections. Changed colors in navbar section to improve readability. Removed padding from form section.

### CODE REVIEW

File: assets/style/style.css [hunk 1]
```diff
@@ -119,37 +117,44 @@ h2 {
...
     text-decoration: none;
     font-size: 1em;
 }
 
+.navigation-list-item:hover a {
+    color: #2F527F;
+    background: #fefefe;
+    font-weight: bold;
+}
+
```
Comment (icelandico) [inline assets/style/style.css]: There's no need to create another hover effect on this element. That's causing the 'bouncy' hover effect.

**Review by icelandico — APPROVED** [review]
Just put the changes from `.navigation-list-item:hover a` class to `.navigation-list-item.hover:hover`

File: assets/style/style.css [hunk 2]
```diff
@@ -119,37 +117,44 @@ h2 {
...
 .navigation-list-item.hover {
     transition: 0.1s linear;
 }
 
 .navigation-list-item.hover:hover {
```
Comment (sebastian-kreft) [inline assets/style/style.css]: You have psuedoclass `hover` so  why did you name normal class `hover`?

Comment (bartzaw) [reply assets/style/style.css]: Hi Sebastian, we are going to rename classes during next week or two, before final review. We realise that there's some classes that we created and need to be refactored. However we need all sections put together to check what class do we got, and what has to be amended.

**Pull request merged**